pymc/tests/test_mixture.py

```diff
@@ -627,53 +632,27 @@ def test_normal_mixture_nd(self, nd, ncomp):
                 Normal.dist(mu=mus[..., i], tau=taus[..., i], shape=nd) for i in range(ncomp)
             ]
             mixture1 = Mixture("m", w=ws, comp_dists=comp_dist, shape=nd)
-            obs1 = Mixture("obs", w=ws, comp_dists=comp_dist, shape=nd, observed=observed)
+            obs1 = Mixture("obs", w=ws, comp_dists=comp_dist, observed=observed)
 
         with Model() as model2:
-            # Expected to fail if comp_shape is not provided,
-            # nd is multidim and it does not broadcast with ncomp. If by chance
-            # it does broadcast, an error is raised if the mixture is given
-            # observed data.
-            # Furthermore, the Mixture will also raise errors when the observed
-            # data is multidimensional but it does not broadcast well with
-            # comp_dists.
```

Comment:
This no longer seems to be a problem. @lucianopaz can you confirm that if the current tests pass, this is indeed fine?